AGENTS.md

```diff
@@ -0,0 +1,330 @@
+# Agent Instructions for www.eclipse.org-eclipse
+
+## Repository Overview
+
+This is the **Eclipse Project Website and New & Noteworthy (N&N) Repository**. It contains the public-facing website content for the Eclipse Platform, SWT, JDT, PDE, and Equinox projects, hosted at https://www.eclipse.org/eclipse/.
+
+**Repository Statistics:**
+- Size: ~318MB, 3,856 files, 352 directories
+- Primary Content: 475 HTML files, 32 Markdown files, 8 shell/Perl scripts
```

Comment:
I know it's merged already but does it really make sense to include these types of statistics which will become out of date very quickly (I don't think anyone will want to maintain that).

Reply:
@danthe1st I don't know bus as this information was automatically derived from the standard template I assume its somehow useful for the AI to get a rough estimate how "complex" a repository is. At least I think it does not harm and we can optimize the instructions any time we thing its sufficient.